src/helpers/login.validations.tsx

```diff
@@ -0,0 +1,14 @@
+const regexPassword = new RegExp('^(?=.*[0-9])(?=.*[a-zA-Z])(?=.+$)');
+
+export const validateEmail = (emailTarget: EventTarget & HTMLInputElement): boolean => {
+    const emailValid: boolean = emailTarget.checkValidity();
```

Comment:
gosto desse artigo a respeito de como nomear variaveis do tipo boolean: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

frasear como se fosse uma pergunta pode deixar o codigo mais claro para entender o proposito da expressao que origina o booleano. O que acha?